src/MedTrackerAPI/Infrastructure/MedTrackerDBContext.cs

```diff
@@ -20,11 +20,11 @@ public record Supply
 public record Device
 {
     public int Id { get; set; }
-    public required string Description { get; set; }
+    public required string? Description { get; set; }
     public string? SerialNumber { get; set; }
-    public required string  Manufacturer { get; set; }
-    public required string  Model { get; set; }
-    public required string  PartNumber { get; set; }
-    public required string  LotNumber { get; set; }
+    public required string?  Manufacturer { get; set; }
```

Comment:
FWIW required and nullable strings don’t have to go together

Reply:
It was the k ly way to get the warnings out of the compiler. I wonder if the NRT warnings could be tweaked though. I actually had it as a note to ask you about it.